<launch>
    <node name = "service_server" pkg = "mastering_ros_demo"
        type = "integer_service_server" output = "screen"/>
    <node name = "service_client" pkg = "mastering_ros_demo"
        type = "integer_service_client" output = "screen"/>
</launch>